### DISCUSSION TITLE
What happened to your blog?

### DISCUSSION THREAD

**Discussion by icanhazbeer** [gimlet-io/gimlet#305] [Q&A] [answered]
The blog is gone!!!
It's just an empty page!

https://gimlet.io/blog

https://gimlet.io/blog/the-cluster-admin-struggle-and-ways-to-keep-kubernetes-resource-requests-and-limits-in-check

Please bring the blog back!

**Comment by laszlocph** [comment, marked as answer]
It is coming back soon.

**Comment by laszlocph** [comment]
It is back! https://gimlet.io/blog
